### PR TITLE
xlink cloud metrics and clean up wording

### PR DESCRIPTION
**Pull request by jessegoodier**
The list of tags is coming up frequently- adding link from cluster detail page.

The cloud-cost-metrics is a bit wordy, this is a first attempt to simpilfy.

### CODE REVIEW

**Comment by jessegoodier** [comment]
@bstuder99 can you verify the `\` usage here? I'm not certain what is/isn't needed.

also feel free to reword, looking to simplify where possible.

**Review by bstuder99 — DISMISSED** [review]
Thanks for cleaning this up
